ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+
+**Engineering/Architecture Lead**
+
+The Developers and SREs do not work in isolation. To obtain the economies of scale that are necessary, and to make sure that work is not duplicated, they need to conform to the 'systems (including processes)' and 'architecture' that best suits the business goal. This is the primary role of this persona. They provide the "guard-rails" for the developers, and help keep the services aligned with accepted strategic technical direction. They also have a key role in developing best practices, and coaching teams to avoid anti-patterns.
+
+**Quality Engineer**
+
+Service development has a life-cycle that ultimately results in a service which is deployed and used by customers. The quality phase of that life-cycle is absolutely critical in the reliability of that service, as SREs need to be able to trust what is being deployed. That is the role for this persona. It is important to note, that this is not the role of the SRE, especially testing in Production environments.
+
+**Site Reliability Engineer (SRE)**
+
+The Site or Service Reliability Engineer is an essential part of the service. Without the SRE, the service is incomplete. Their primary focus is ensuring that the services are "running". When there is a problem, it is up to them to resolve the issue as quickly as possible. However, this is only the end goal. The role of an SRE is to contribute to the reliability of the service so that the need to fix something is the exception rather than the rule. This is best epitomized in this quote by one of Red Hat's SRE Managers: "Do not accept repeated failure". That is, if something fails (e.g. SRE is alerted), then follow through with all "actions" so that the problem does not repeat. These actions can be things like: from identifying the issue with the responsible engineering team through to actually fixing the problem in code, personally. 
+
+In many organizations, SRE functions support more than one service. Many projects undertaken by SRE teams provide the tooling, workflows and platforms needed to operate at scale. A key metric is for the SRE team to grow sub-linearly with "hockey stick" customer growth while minimizing toil and technical debt.
+
+SREs are also valuable consultants, particularly in these key areas: observability; automation; resilience; toil prevention; capacity planning; and, operability. They are most familiar with their operational boundaries, and can best articulate the problems being encountered since they are hands-on.
+
+**Support Agent**
+
+This persona provides assistance to Service Consumers, usually in the form of support cases raised by customers. They may require elevated levels of access to the service internals in order to help consumers, but in a way that ensures they cannot degrade the service itself. 
```

Comment:
Support has similar hands-on access to observability, resilience, toil detection/prevention, etc. to SRE.  QE feedback is "hey, devs, when I hold your thing like `$THIS` it breaks like `$THIS`".  Support and SRE are both well positioned for "hey, devs, your thing is breaking like `$THIS` all. the. time.  Like, twice as often as all the other failure modes put together.  Please fix :)" and for "hey, devs, when your thing breaks, it's really hard to figure out why and repair.  Please improve observability over `$HERE`.

Reply:
Also, they are Support Engineers, not Support Agents.

Reply:
@wking as you mentioned above also, I will add that they are invaluable contributors to improving the reliability. I will do so for the QE folks also.

Reply:
Fixed